ocs_ci/ocs/resources/ocs.py

```diff
@@ -255,3 +255,45 @@ def check_if_cluster_was_upgraded():
 
     """
     return True if "replaces" in get_ocs_csv().get().get("spec") else False
+
+
+def get_all_jobs(namespace=defaults.ROOK_CLUSTER_NAMESPACE):
```

Comment:
as discussed, please move this function to a new `job.py` file

Reply:
Okay, that will do.